invenio_rdm_records/resources/serializers/csv/schema.py

```diff
@@ -0,0 +1,68 @@
+from commonmeta import doi_as_url
+from invenio_communities import current_communities
+from invenio_communities.communities.services.service import get_cached_community_slug
+from marshmallow import Schema, fields, missing
+from marshmallow_utils.fields import SanitizedHTML, SanitizedUnicode
+
+
+class CSVSchema(Schema):
+    id = SanitizedUnicode(attribute="")
+    title = SanitizedUnicode(attribute="metadata.title")
+    description = SanitizedHTML(attribute="metadata.description")
+    access_right = fields.Method("get_access")
+    # upload_type = SanitizedUnicode(attribute="")
+    # image_type = SanitizedUnicode(attribute="")
+    communities = fields.Method("get_communities")
+    # publication_date = SanitizedUnicode(attribute="")
+    # journal_title = SanitizedUnicode(attribute="")
```

Comment:
Since the code for custom fields is commented out, this comment can be shelved (just sharing some knowledge for future developments):

for custom fields, we typically use processors like [this](https://github.com/inveniosoftware/invenio-rdm-records/blob/29db5c278184314bf7e3b35a8001b083492775a0/invenio_rdm_records/resources/serializers/dublincore/__init__.py#L30)

Reply:
oh yeah this schema.py file can be deleted as it is not used anywhere